### PR TITLE
update pos creator tests with realistic values

### PR DESCRIPTION
**Pull request by dorin-iancu**

### CODE REVIEW

**Review by alyn509 — APPROVED** [review]

File: auto-pos-creator/tests/pos_creator_setup/mod.rs [hunk 1]
```diff
@@ -67,6 +67,20 @@ where
...
+    // B:C 1:3
+
+    // Pools: (B = billion)
+    // (A, B) => (1B, 2B)
+    // (A, C) => (1B, 6B)
+    // (B, C) => (1B, 3B)
+    //
+    // A_total = 2B
+    // B_total = 3B
+    // C_total = 9B
```
Comment (psorinionut) [inline auto-pos-creator/tests/pos_creator_setup/mod.rs:83]: I think here it would be C_total = 6B?

Comment (dorin-iancu) [reply auto-pos-creator/tests/pos_creator_setup/mod.rs:83]: Nope, 6B from the (A, C) pool + 3B from (B, C) pool

Comment (psorinionut) [reply auto-pos-creator/tests/pos_creator_setup/mod.rs:83]: Something still seems off. I don't really understand to what you are referring to with those numbers.

Comment (dorin-iancu) [reply auto-pos-creator/tests/pos_creator_setup/mod.rs:83]: Total number of tokens, so I know the total overall balance I'll need to set on line 133 for the owner's balance.

**Review by psorinionut — APPROVED** [review]

**Pull request merged**